app/assets/javascripts/courseEdit/components/CourseEditPage.js

```diff
@@ -65,6 +67,20 @@ class CourseEditPage extends React.Component {
     this.updateCourse(params)
   }
 
+  onImage(value) {
+    const params = {
+      course: {
+        photo_attributes: {
+          image_data: value
+        },
+      },
+    }
+    const path = APIRoutes.editCoursePath(this.id)
+    request.post(path, params, (response) => {
```

Comment:
this should be a put - i think you might actually have been creating new courses haha